<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="17" passed="13">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="14380" started-at="2018-07-27T11:40:04Z" finished-at="2018-07-27T11:40:18Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="14380" started-at="2018-07-27T11:40:04Z" finished-at="2018-07-27T11:40:18Z">
      <class name="api.PostUsers">
        <test-method status="PASS" signature="postUserTest()[pri:0, instance:api.PostUsers@38082d64]" name="postUserTest" duration-ms="372" started-at="2018-07-27T14:40:18Z" finished-at="2018-07-27T14:40:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postUserTest -->
      </class> <!-- api.PostUsers -->
      <class name="api.GetUsers">
        <test-method status="PASS" signature="getAllUsersIdEach()[pri:0, instance:api.GetUsers@2ed94a8b]" name="getAllUsersIdEach" duration-ms="5250" started-at="2018-07-27T14:40:10Z" finished-at="2018-07-27T14:40:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllUsersIdEach -->
        <test-method status="PASS" signature="getUsersRestAssuredTest()[pri:0, instance:api.GetUsers@2ed94a8b]" name="getUsersRestAssuredTest" duration-ms="368" started-at="2018-07-27T14:40:16Z" finished-at="2018-07-27T14:40:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUsersRestAssuredTest -->
        <test-method status="PASS" signature="getUsersTest()[pri:0, instance:api.GetUsers@2ed94a8b]" name="getUsersTest" duration-ms="1181" started-at="2018-07-27T14:40:16Z" finished-at="2018-07-27T14:40:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUsersTest -->
        <test-method status="PASS" signature="getUsersTestNGTest()[pri:0, instance:api.GetUsers@2ed94a8b]" name="getUsersTestNGTest" duration-ms="312" started-at="2018-07-27T14:40:17Z" finished-at="2018-07-27T14:40:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUsersTestNGTest -->
      </class> <!-- api.GetUsers -->
      <class name="api.AutenticatTest">
        <test-method status="PASS" signature="noAuthenticationTest()[pri:0, instance:api.AutenticatTest@79b4d0f]" name="noAuthenticationTest" duration-ms="1473" started-at="2018-07-27T14:40:04Z" finished-at="2018-07-27T14:40:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- noAuthenticationTest -->
        <test-method status="PASS" signature="successAuthenticationTest()[pri:0, instance:api.AutenticatTest@79b4d0f]" name="successAuthenticationTest" duration-ms="477" started-at="2018-07-27T14:40:05Z" finished-at="2018-07-27T14:40:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- successAuthenticationTest -->
      </class> <!-- api.AutenticatTest -->
      <class name="api.CreateUserTest">
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="685" started-at="2018-07-27T14:40:06Z" data-provider="New Users" finished-at="2018-07-27T14:40:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ion Creanga]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[writer]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="522" started-at="2018-07-27T14:40:06Z" data-provider="New Users" finished-at="2018-07-27T14:40:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Mihai Eminescu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[poet]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="523" started-at="2018-07-27T14:40:07Z" data-provider="New Users" finished-at="2018-07-27T14:40:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Simona Halep]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tennis player]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="341" started-at="2018-07-27T14:40:07Z" data-provider="New Users" finished-at="2018-07-27T14:40:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Klaus Iohannis]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[president]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="419" started-at="2018-07-27T14:40:08Z" data-provider="New Users" finished-at="2018-07-27T14:40:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Giorgiana Ghilea]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[trainee]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="FAIL" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="646" started-at="2018-07-27T14:40:08Z" data-provider="New Users" finished-at="2018-07-27T14:40:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[012345678910111213]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[job1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at api.CreateUserTest.methodCreateUserTest(CreateUserTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="PASS" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="364" started-at="2018-07-27T14:40:09Z" data-provider="New Users" finished-at="2018-07-27T14:40:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[vasile]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[portar]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="FAIL" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="340" started-at="2018-07-27T14:40:09Z" data-provider="New Users" finished-at="2018-07-27T14:40:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[vasile]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[portar]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at api.CreateUserTest.methodCreateUserTest(CreateUserTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="FAIL" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="544" started-at="2018-07-27T14:40:09Z" data-provider="New Users" finished-at="2018-07-27T14:40:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[nume2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11111111111111111111111111111111111]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at api.CreateUserTest.methodCreateUserTest(CreateUserTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
        <test-method status="FAIL" signature="methodCreateUserTest(java.lang.String, java.lang.String, int)[pri:0, instance:api.CreateUserTest@73f792cf]" name="methodCreateUserTest" duration-ms="523" started-at="2018-07-27T14:40:10Z" data-provider="New Users" finished-at="2018-07-27T14:40:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at api.CreateUserTest.methodCreateUserTest(CreateUserTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodCreateUserTest -->
      </class> <!-- api.CreateUserTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
